apps/jenkins/jenkins/sbox-intsvc/jenkins.yaml

```diff
@@ -7,6 +7,14 @@ spec:
   values:
     checkDeprecation: false
     controller:
+      healthProbes: true
+      probes:
+        startupProbe:
+          httpGet:
+            path: '/login'
+            port: http
```

Comment:
Not sure if port needs explicitly declared here

Reply:
can you try by just overriding what you want to override rather than specifying the defaults